lib/src/sv_onvif.h

```diff
@@ -35,6 +39,26 @@ typedef enum {
   OMS_AUTHENTICATION_ERROR = -30,
   OMS_UNKNOWN_FAILURE = -100
 } MediaSigningReturnCode;
+
+// Dummy function to prevent compilation errors
+static inline MediaSigningReturnCode
+onvif_media_signing_set_signing_key_pair(onvif_media_signing_t *self,
+    const char *private_key,
+    size_t private_key_size,
+    const char *certificate_chain,
+    size_t certificate_chain_size,
+    bool user_provisioned)
+{
+
+  (void)self;  // Prevent unused variable warnings
```

Comment:
Have a look in sv_internal.h
There is a  define for unused variable that I think is a better option. ATTR_UNUSED
Put that in a good place so it can be included and used here as well.